index.html

```diff
@@ -408,7 +408,7 @@ <h1>Characters and strings</h1>
                </tr>
                <tr>
                  <td>All occurrences</td>
-                 <td><code>matchall(pat, str)</code></td>
+                 <td><code>collect((m.match for m = eachmatch(pat, str)))</code></td>
```

Comment:
I suggest `[m.match for m in eachmatch(pat, str)]` in stead.

Reply:
Agree, thats simpler